packages/templates/clients/websocket/test/microcks-setup/microcks-podman.yml

```
@@ -173,6 +173,18 @@ services:
+  tester-java:
+    profiles:
+      - test-java
+    image: maven:3.9-eclipse-temurin-21
+    container_name: websocket-acceptance-tester-java
+    ports:
+      - "8084:8084"
+    working_dir: /usr/src/app
+    volumes:
+      - ../../../../../../:/usr/src/app
+    command: ["sh", "-c", "cd packages/templates/clients/websocket/java/quarkus/test/temp/snapshotTestResult/client_hoppscotch && mvn clean install && cd ../../../../../../test/java/quarkus && mvn clean test"]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)